contracts/modules/src/pause.rs

```diff
@@ -48,6 +48,12 @@ pub trait PauseModule {
         require!(self.not_paused(), "Contract is paused");
     }
 
+    #[event("pause_contract")]
```

Comment:
i think the event is usually cammelCase

Reply:
in the sdk there are examples with both cases, do you prefer camelCase for this one?